lib/io_page.mli

```diff
@@ -35,19 +36,27 @@ val get_addr : t -> nativeint
 val get_page : t -> nativeint
 (** [get_page t] returns the page offset (get_addr t) mod page_size, starting at 0 .*)
 
-val get : int -> t
+val get : ?n:int -> unit -> t
```

Comment:
why make this `?n` an optional keyword? Is it worth the trouble? what is the default value for `n`?

Reply:
Thank you @hannesm . I can't recall why I've set that, but I agree I had multiple places to just add the parameter's name. I currently think it's fine to leave the function's type unchanged. (The default value was 1 as it was the most used value).